p4-fusion/commands/change_list.cc

```diff
@@ -6,27 +6,27 @@
  */
 #include "change_list.h"
 
+#include <utility>
+
 #include "p4_api.h"
 #include "describe_result.h"
 #include "filelog_result.h"
 #include "print_result.h"
 #include "utils/std_helpers.h"
 #include "minitrace.h"
 
-#include "thread_pool.h"
-
-ChangeList::ChangeList(const std::string& clNumber, const std::string& clDescription, const std::string& userID, const int64_t& clTimestamp)
-    : number(clNumber)
-    , user(userID)
-    , description(clDescription)
+ChangeList::ChangeList(std::string  clNumber, std::string  clDescription, std::string  userID, const int64_t& clTimestamp)
```

Comment:
Functions and constructors should generally not take in `std::string` as an argument. If you mean to transfer ownership, use `std::string &&`. If you mean to only let the callee borrow the string for a bit, using `const std::string &`.

```suggestion
ChangeList::ChangeList(std::string &&number, std::string &&description, std::string &&userID, int64_t timestamp)
```

The caller can create a copy if needed, or use `std::move` to forward in strings from elsewhere.